doc/source/methods/ALE.ipynb

```diff
@@ -0,0 +1,285 @@
+{
```

Comment:
<ul><li>I think you may have forgotten the space before/after $$\hat{M}(x_1)$$ as this does not render well ... Potentially as blank line is missing after $$M(x_1) = ...$$.</li><li>So the estimation of the conditional tells us to fix $x_1$ and then check for some instances that have a similar enough $x_1$. How do we select N(x1) in practice? </li><li>Esentially, what we are saying is that fixing x1 to some value changes the distribution of x2 (which could we see from the plots below), and that this biases M(x1), right?</li><li>Maybe worth stating this [_ Furthermore, suppose height doesn't actually have any effect on the prediction, only weight does_] mathematically as the case where the function _f(x_1, x_2) = x_1_ has _M(x_2) \neq 0_ which is not intuitive. </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<ul><li>Added blank lines although it rendered fine for me</li><li>Estimating conditional distributions is a hard problem in general, in practice this is done by binning data using small enough quantiles (but big enough so that <code>min_bin_points</code>  is satisfied. I mention this in the estimation formula: "Here z0, z1,... is a sufficiently fine grid..."</li><li>It's not the change of the distribution of x2 that biases M(x1) but rather that M(x1) cannot distinguish between effects of x1 and effects of other features correlated with x2</li><li>Stated!</li></ul>